### PR TITLE
Automatically port System.Text.Json.Serialization.JsonConverterAttribute

### PR DESCRIPTION
**Pull request by carlossanlop**
New batch.

### CODE REVIEW

File: xml/System.Text.Json.Serialization/JsonConverterAttribute.xml [hunk 1]
```diff
@@ -102,8 +114,8 @@
       </Parameters>
       <Docs>
         <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>If overridden and <see cref="P:System.Text.Json.Serialization.JsonConverterAttribute.ConverterType" /> is <see langword="null" />, allows a custom attribute to create the converter in order to pass additional state.</summary>
```
Comment (Jozkee) [inline xml/System.Text.Json.Serialization/JsonConverterAttribute.xml]: @ahsonkhan when you say "custom" you mean a class that implements JsonConverterAttribute?
```suggestion
        <summary>When overridden in a derived class and <see cref="P:System.Text.Json.Serialization.JsonConverterAttribute.ConverterType" /> is <see langword="null" />, allows the derived class to create a <see cref="T:System.Text.Json.Serialization.JsonConverter"/> in order to pass additional state.</summary>
```

File: xml/System.Text.Json.Serialization/JsonConverterAttribute.xml [hunk 2]
```diff
@@ -77,7 +89,7 @@
         <ReturnType>System.Type</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The type of the converter to create, or <see langword="null" /> if <see cref="M:System.Text.Json.Serialization.JsonConverterAttribute.CreateConverter(System.Type)" /> should be used to obtain the converter.</summary>
         <value>To be added.</value>
```
Comment (Jozkee) [inline xml/System.Text.Json.Serialization/JsonConverterAttribute.xml]:
```suggestion
         <value>The type of the <see cref="T:System.Text.Json.Serialization.JsonConverterAttribute"/>, or <see langword="null" /> if it was created without a type.</value>
```

File: xml/System.Text.Json.Serialization/JsonConverterAttribute.xml [hunk 3]
```diff
@@ -77,7 +89,7 @@
         <ReturnType>System.Type</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The type of the converter to create, or <see langword="null" /> if <see cref="M:System.Text.Json.Serialization.JsonConverterAttribute.CreateConverter(System.Type)" /> should be used to obtain the converter.</summary>
```
Comment (Jozkee) [inline xml/System.Text.Json.Serialization/JsonConverterAttribute.xml]: I really do not understand this paragraph.
I think it should say something like this:
```suggestion
        <summary>Gets the type of the <see cref="T:System.Text.Json.Serialization.JsonConverterAttribute"/>, or <see langword="null" /> if it was created without a type.</summary>
```

File: xml/System.Text.Json.Serialization/JsonConverterAttribute.xml [hunk 4]
```diff
@@ -19,8 +19,20 @@
...
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The specified converter type must derive from <xref:System.Text.Json.Serialization.JsonConverter>.
+
+When placed on a property, the specified converter will always be used.
+
+When placed on a type, the specified converter will be used unless a compatible converter is added to <xref:System.Text.Json.JsonSerializerOptions.Converters> or there is another <xref:System.Text.Json.Serialization.JsonConverterAttribute> on a property of the same type.
```
Comment (rpetrusha) [inline xml/System.Text.Json.Serialization/JsonConverterAttribute.xml]:
```suggestion
When placed on a type, the specified converter will be used unless a compatible converter is added to the <xref:System.Text.Json.JsonSerializerOptions.Converters?displayProperty=nameWithType> collection or there is another <xref:System.Text.Json.Serialization.JsonConverterAttribute> on a property of the same type.
```

File: xml/System.Text.Json.Serialization/JsonConverterAttribute.xml [hunk 5]
```diff
@@ -102,10 +114,10 @@
       </Parameters>
       <Docs>
         <param name="typeToConvert">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Text.Json.Serialization/JsonConverterAttribute.xml]:
```suggestion
        <param name="typeToConvert">The type of the converter.</param>
```

**Comment by rpetrusha** [comment]
I'll merge this PR now, @carlossanlop.

**Pull request merged**